RecoJets/JetProducers/plugins/BoostedTauSeedsProducer.cc

```diff
@@ -159,23 +159,47 @@ namespace
     }
   }
 
-  std::vector<reco::PFCandidateRef> getPFCandidates_exclJetConstituents(const edm::Handle<reco::PFCandidateCollection>& pfCandidates, const reco::Jet::Constituents& jetConstituents, double dRmatch, bool invert)
-  {
+  // std::vector<reco::PFCandidateRef> getPFCandidates_exclJetConstituents(const edm::Handle<reco::PFCandidateCollection>& pfCandidates, const reco::Jet::Constituents& jetConstituents, double dRmatch, bool invert)
+  // {
+  //   std::vector<reco::PFCandidateRef> pfCandidates_exclJetConstituents;
+  //   size_t numPFCandidates = pfCandidates->size();
+  //   for ( size_t pfCandidateIdx = 0; pfCandidateIdx < numPFCandidates; ++pfCandidateIdx ) {
+  //     reco::PFCandidateRef pfCandidate(pfCandidates, pfCandidateIdx);
+  //     bool isJetConstituent = false;
+  //     for ( reco::Jet::Constituents::const_iterator jetConstituent = jetConstituents.begin();
+  // 	    jetConstituent != jetConstituents.end(); ++jetConstituent ) {
+  // 	double dR = deltaR(pfCandidate->p4(), (*jetConstituent)->p4());
+  // 	if ( dR < dRmatch ) {
+  // 	  isJetConstituent = true;
+  // 	  break;
+  // 	}
+  //     }
+  //     if ( !(isJetConstituent^invert) ) {
+  // 	pfCandidates_exclJetConstituents.push_back(pfCandidate);
+  //     }
+  //   }
+  //   return pfCandidates_exclJetConstituents;
+  // }
```

Comment:
Why are you keeping this old, commented-out code? If there is a reason to keep it, add a comment explaining why it needs to be kept. Otherwise delete it.